users/views.py

```diff
@@ -178,6 +178,11 @@ def get(self, request, *args, **kwargs):
                 # validated again in the parent get method.
                 pass
 
+        # Keycloak action passthrough
+        if request.GET.get('kc_action'):
+            print(request.GET.get('kc_action'))
```

Comment:
is this some leftover from debugging? Could be removed or changed to be outputted via logger.